### PR TITLE
Implement functionality

### PR DESCRIPTION
**Pull request by maherramadan78**
### This feature branch holds the following:
Added index.html
Added styles.css
Added gitignore
Added linters
Added readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,62 @@
...
+## Live Demo (if available)
+
+[Live Demo Link](https://livedemo.com)
+
+
+## Getting Started
+
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+
```
Comment (zieeco) [inline README.md]: - Please, kindly replace this text with some instructions on how users can have your project locally on their machine. These could include the `Prerequisite`, `Setup`, etc to make your readme more professional as this text was added for you as a guideline for you to follow to update the "Getting Started" section of your readme.

File: README.md [hunk 2]
```diff
@@ -1 +1,62 @@
...
+### Install
+
+### Usage
+
+### Run tests
+
+### Deployment
+
+
```
Comment (zieeco) [inline README.md]: - You could remove these headers as they're not useful for now for this project, however you'd add them for the subsequent project while you make progress in the program.

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @maherramadan78,


Great job so far in setting up the Hello Microverse Project 👏 
You've done it well, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Good beginnign. ✔️
- Descriptive PR. ✔️


## Required Changes ♻️

- [ ] Please, kindly update the title of your PR to be more descriptive as you've not added any functionality to this project. You could update it to `Hello Microvese Project`, or `Greetings` as this best describes the project, and gives an overview of what you've done in the PR.

_Check the comments under the review._





## Optional suggestions
- N/A

**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (maherramadan78) [reply README.md]: thanks a lot , i appreciate.
have a wonderful day

File: README.md (hunk 2 above)
Comment (maherramadan78) [reply README.md]: ok i will check it out , thank you so much

**Comment by maherramadan78** [comment]
Hello , kindly , could you please describe to me what i need to do over here with this Project.
Thanks you so much for your attention.
Have a wonderful day.

Enviado desde Correo<https://go.microsoft.com/fwlink/?LinkId=550986> para Windows

De: ***@***.***>
Enviado: miércoles, 21 de septiembre de 2022 12:30 p. m.
Para: ***@***.***>
CC: ***@***.***>; ***@***.***>
Asunto: Re: [maherramadan78/Microverse-project] Implement functionality (PR #1)


@zieeco requested changes on this pull request.

Hi @maherramadan78<https://github.com/maherramadan78>,

Great job so far in setting up the Hello Microverse Project 👏
You've done it well, however, there are some issues that you still need to work on to go to the next project, but you are almost there!

To highlight 🎯

  *   No linter errors.✔️
  *   Good beginnign. ✔️
  *   Descriptive PR. ✔️

Required Changes ♻️

  *   [ ] Please, kindly update the title of your PR to be more descriptive as you've not added any functionality to this project. You could update it to Hello Microvese Project, or Greetings as this best describes the project, and gives an overview of what you've done in the PR.

Check the comments under the review.

[https://camo.githubusercontent.com/aa9137eba4285ba68320a19592e5e655279c09be5c8fc2fb7ef499d61cf26010/68747470733a2f2f6d65646961302e67697068792e636f6d2f6d656469612f3855475651455370444d4d516c37796433462f67697068792e676966]<https://camo.githubusercontent.com/aa9137eba4285ba68320a19592e5e655279c09be5c8fc2fb7ef499d61cf26010/68747470733a2f2f6d65646961302e67697068792e636f6d2f6d656469612f3855475651455370444d4d516c37796433462f67697068792e676966>

Optional suggestions

  *   N/A

Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.


As described in the Code reviews limits policy<https://microverse.zendesk.com/hc/en-us/articles/1500004088561> you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this form<https://airtable.com/shrQAqnBwek5a0O0s>.


In README.md<https://github.com/maherramadan78/Microverse-project/pull/1#discussion_r976802616>:

> +**This is an example of how you may give instructions on setting up your project locally.**



+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**



+



  *   Please, kindly replace this text with some instructions on how users can have your project locally on their machine. These could include the Prerequisite, Setup, etc to make your readme more professional as this text was added for you as a guideline for you to follow to update the "Getting Started" section of your readme.


In README.md<https://github.com/maherramadan78/Microverse-project/pull/1#discussion_r976804226>:

> +### Install



+



+### Usage



+



+### Run tests



+



+### Deployment



+



+



  *   You could remove these headers as they're not useful for now for this project, however you'd add them for the subsequent project while you make progress in the program.

**Review by nicupop729 — COMMENTED** [review]
Hello @maherramadan78 👋,

_This is an additional review as a result of the second opinion request._

Kudos for the good work you have done on this project! :clap:

I also find the comment of the code reviewer from the summary regarding the name of the PR redundant as you are implementing the functionality of the app through this branch.
Please don't take that comment into consideration, but only those about README.
Anyways, I noticed that you already updated your README.

That should be fine already. 
You can ask for another review in the dashboard.

Keep rocking 🚀!

**Review by nedjwab — APPROVED** [review]
# 🟢 Project Approved! 🏆

## Hi @maherramadan78   👋

Your project is complete! ✔️ 🏁

Your code was very professional 💯

There is nothing else to say other than... _it's time to merge it_ :shipit:
Congratulations! 🎉

 
## Highlights 🌟

- ✔️ Descriptive commmit messsages
- ✔️ Descriptive PR description
- ✔️ No linter Errors
- The code is top-notch! 💯 and have a huge potential.

Cheers and Happy coding!👏👏👏

## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.


_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by maherramadan78** [comment]
I'm repeating my block 
This is my hello microverse project.

**Review by zieeco — COMMENTED** [review]
# status: APPROVED :green_circle: :100: 

Hi, @maherramadan78, 


68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f78543078657a514755357843444a754350652f67697068792e676966


Good job so far 👍 Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights ⚡

✅ Descriptive PR 👍 
✅ Good commit messages 👍 
✅ Linters are passing 👏 


## Optional suggestions
- N/A


_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**